Cargo.lock

```diff
@@ -4,18 +4,18 @@ version = 3
 
 [[package]]
 name = "aho-corasick"
-version = "1.0.5"
+version = "1.1.1"
```

Comment:
I would suggest updating just vhost and vhost-user-backend: `cargo update -p vhost -p vhost-user-backend`, but we can fix later ;-)

Thanks for this PR!

Reply:
> I would suggest updating just vhost and vhost-user-backend: `cargo update -p vhost -p vhost-user-backend`, but we can fix later ;-)

I would add `--aggressive` to also update whatever those crates depend on, but yeah. I will clean that once the release is there.